app/mm-redux/actions/posts.ts

```diff
@@ -210,6 +210,9 @@ export function createPost(post: Post, files: any[] = []) {
             type: PostTypes.RECEIVED_NEW_POST,
             data: {
                 ...newPost,
+                props: { // Will be overwritten with server response
+                    disable_group_highlight: true,
```

Comment:
This looks odd to me, is there another way to accomplish it? I do not expect to set some random state for props that get added and then have to keep on modifying this

Reply:
I felt adding here might make the object structure consistent with the server response but yes, we can make changes in the component level as well and check for the existence of the property.